<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourierCompanyWindowClass</class>
 <widget class="QWidget" name="CourierCompanyWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CourierCompanyWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="packages_list"/>
   </item>
   <item>
    <widget class="QLineEdit" name="recipient_line_edit">
     <property name="placeholderText">
      <string>Give the recipient...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="street_line_edit">
     <property name="placeholderText">
      <string>Street</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="number_line_edit">
     <property name="placeholderText">
      <string>Number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="x_line_edit">
     <property name="placeholderText">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="y_line_edit">
     <property name="placeholderText">
      <string>Y</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="add_push_button">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
